src/se/terhol/pisemka31/IceCream.java

```diff
@@ -56,4 +56,9 @@ public boolean equals(Object obj) {
     public int hashCode() {
         return this.nameCz.hashCode();
     }
+
+    @Override
+    public int compareTo(Object o) {
```

Comment:
Once you add a generic type to a interface definition, here you can specify the type as well

Reply:
Done